scripts/semconv/templates/registry/stable/weaver.yaml

```diff
@@ -1,6 +1,7 @@
 
 params:
-  excluded_attributes: ["messaging.client_id"]
+  # Exclude `network.interface.name` for now because https://github.com/open-telemetry/semantic-conventions/issues/1777
+  excluded_attributes: ["messaging.client_id", "network.interface.name"]
```

Comment:
👍 

we're fixing this attribute and it should not happen again, but you can also steal a more general workaround from https://github.com/open-telemetry/opentelemetry-python/pull/4391 

see my other comment

Reply:
Ooh, thanks. Will try that.

Reply:
commit 3b786b19d. Thanks.